decidim-elections/app/packs/src/decidim/votings/polling_officer_zone/sign-closure.js

```diff
@@ -0,0 +1,23 @@
+$(() => {
+  const $submitBtn = $("#btn-submit-from-modal");
+  const $modalBtn = $("#btn-modal-closure-sign");
+  const $signCheckbox = $("#closure_sign_signed");
+
+  const changeButtonProps = () => {
+    $modalBtn.toggleClass("disabled");
+    $submitBtn.toggleClass("disabled");
+
+    if ($signCheckbox.is(":checked")) {
+      $submitBtn.prop("disabled", false);
+      $modalBtn.prop("disabled", false);
+    } else {
+      $submitBtn.prop("disabled", true);
+      $modalBtn.prop("disabled", true);
+    }
+  };
+
+  $modalBtn.addClass("disabled");
+  $submitBtn.addClass("disabled");
```

Comment:
💬   The "disabled" class, along with the "disabled" attribute/property, can be added to the HTML template in order to avoid the glitch when rendering the page (button enabled until the javascript disables it).